asn.1/message.asn1

```diff
@@ -53,7 +53,8 @@ PIdentity ::= SEQUENCE {
     user-id     PString,
     username    PString OPTIONAL,
     comm-type   INTEGER (0..255),
-    lang        ISO639-1 OPTIONAL
+    lang        ISO639-1 OPTIONAL,
+    flags       INTEGER (0..61440)
```

Comment:
0xf000? Please use at least full 16 bits, that is 65535, easier to understand, and no risk of the asn1 code rejecting a valid pattern.

Reply:
ok, I had just based it in the max possible value for the number of hexadecimal digits used.